frontend/src/features/contacts/contact/detail/Organization.test.tsx

```diff
@@ -187,13 +187,14 @@ describe('Contact OrganizationView component', () => {
     });
 
     var phoneValueElements = component.getAllByTestId('phone-value');
-    expect(phoneValueElements.length).toBe(4);
+    expect(phoneValueElements.length).toBe(5);
```

Comment:
@FuriousLlama this is what I mean, there are 5 contact methods in this test but previously only 4 were displayed. Also, the order that they were displayed in was different.

Reply:
Hoping you remember if there were requirements for that, if not I can try and dig through the initial story.